### PR TITLE
21 add latex to template

### PR DESCRIPTION
**Pull request by SimonFreyaldenhoven**
On this branch I added compilation of the tex file to the pipeline.

@kobleary  - could you check if `main.py` runs successfully for you and do a code review?

### CODE REVIEW

File: analysis/source/lib/pyHelper.py [hunk 1]
```diff
@@ -36,18 +39,26 @@ def run_script(script, folder, absolute_path = os.getcwd(), program = "python",
...
+        p = subprocess.run(command, capture_output = True) 
+        toc = timer()
+    else: #traverse to products folder
+        tic = timer()
+        full_path = os.path.join(products_loc, folder, script)
+        os.chdir(os.path.join(products_loc, folder))
+        command = program +' '+ script
+        p = subprocess.run(command)
+        p = subprocess.run('bibtex ' + script.strip('.tex')) # .tex file without extension 
+        p = subprocess.run(command)
```
Comment (kobleary) [inline analysis/source/lib/pyHelper.py:57]: @SimonFreyaldenhoven Why do we have several `p = subprocess.run(command)` lines?

Comment (SimonFreyaldenhoven) [reply analysis/source/lib/pyHelper.py:57]: @kobleary this is needed to get all the internal references right. (e.g. the citations and the numbers in things like Figure 2). It usually happens automatically in the background in your latex distribution.

Without any type of \ref{} command, and no citations, you would only need to do it once.

**Comment by SimonFreyaldenhoven** [comment]
@kobleary can you try again? In the commit above, I changed the file format to png, which should be more robust than .eps

File: analysis/source/lib/pyHelper.py (hunk 1 above)
Comment (kobleary) [reply analysis/source/lib/pyHelper.py:57]: @SimonFreyaldenhoven ok got it! thanks

**Pull request merged**